### PR TITLE
Add CocoaPods support

### PR DESCRIPTION
**Pull request by ReadmeCritic**
This is a starter .podspec file for CocoaPods support

Left to do
- [ ] get setup with CocoaPods Trunk service (to publish/make it public) @timonus https://guides.cocoapods.org/making/getting-setup-with-trunk.html
- [x] update README

### CODE REVIEW

File: TJDropbox.podspec [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  s.license      = "BSD 3-clause \"New\" or \"Revised\" License"
+
+  s.author       = "Tim Johnsen"
+
+  s.source       = { :git => "https://github.com/timonus/TJDropbox.git", :tag => "0.4" }
+
+  s.source_files = "TJDropbox/*.{h,m}"
+
+  s.ios.deployment_target = "8.0"
+  # s.osx.deployment_target = "10.9"
```
Comment (timonus) [inline TJDropbox.podspec]: TJDropbox actually already supports macOS 😄 but TJDropboxAuthenticationViewController.h/m needs to be excluded. I'm not familiar with CocoaPods, but could you update to add that?

**Comment by timonus** [comment]
Thanks for your contribution! Left one small comment. Happy to merge once that's handled.

**Comment by timonus** [comment]
(Also, if you'd like to update the README go for it. If not, I'm happy to).

**Comment by ReadmeCritic** [comment]
@timonus updated for `osx`

**Comment by ReadmeCritic** [comment]
For the README, was thinking an _Installation_ section above _Migrating..._
Is that ok?

**Comment by timonus** [comment]
@ReadmeCritic I think that would be ok as long as it's brief. I may change it at a later time.

**Comment by ReadmeCritic** [comment]
@timonus ok

last but not least, the trunk service. let me know if you need help

**Comment by timonus** [comment]
Thank you so much for doing this! Will work on adding to the CocoaPods trunk.

**Comment by timonus** [comment]
Alright @ReadmeCritic, I _believe_ this is now published to CocoaPods.

**Pull request merged**